static/skin/i18n/el.json

```diff
@@ -4,7 +4,7 @@
 			"Norhorn"
 		]
 	},
-	"welcome-page-overzealous-filter": "Κανένα αποτέλεσμα. Θέλετε να <a href=\"?lang=\">επαναφέρετε το φίλτρο</a>;",
+	"welcome-page-overzealous-filter": "Κανένα αποτέλεσμα. Θέλετε να <a href=\"{{URL}}=\">επαναφέρετε το φίλτρο</a>;",
```

Comment:
This is a wrong update to the syntax - there is an extra `=` after `{{URL}}`. In `en.json` the link tag portion underwent the following transformation

- `<a href=\"?lang=\">` → `<a href=\"#lang=\">` → `<a href=\"{{URL}}\">`

which turned out to be confusing for the translators since this error is repeated in a number of other translations (dga, he, ia, etc (the latter standing for "et cetera", rather than denoting a language code :smile:)).

I was told that translation files shouldn't be updated in our repository (because any such updates will be overwritten by Translatewiki). So how do we fix these errors?

Reply:
The translation is managed here : https://translatewiki.net/w/i.php?title=Special:Translate&group=kiwix-desktop&language=fr&filter=&action=translate

But I haven't found where to open a issue. @kelson42 may know better than me.

Reply:
We need to fix the "bug" in the translation upstream, so in TW.

I have done it at:
* https://translatewiki.net/w/i.php?title=Kiwix%3AServe-welcome-page-overzealous-filter%2Fel&diff=11554708&oldid=11507609
* https://translatewiki.net/w/i.php?title=Kiwix%3AServe-welcome-page-overzealous-filter%2Fdga&diff=11554713&oldid=11507599
* https://translatewiki.net/w/i.php?title=Kiwix%3AServe-welcome-page-overzealous-filter%2Fia&diff=11554717&oldid=11507610

@veloman-yunkan My recommendation, is please open an account on TW so you can do it yourself when needed and possible.

Reply:
> We need to fix the "bug" in the translation upstream, so in TW.
> 
> I have done it at:
> 
>   * https://translatewiki.net/w/i.php?title=Kiwix%3AServe-welcome-page-overzealous-filter%2Fel&diff=11554708&oldid=11507609
> 
>   * https://translatewiki.net/w/i.php?title=Kiwix%3AServe-welcome-page-overzealous-filter%2Fdga&diff=11554713&oldid=11507599
> 
>   * https://translatewiki.net/w/i.php?title=Kiwix%3AServe-welcome-page-overzealous-filter%2Fia&diff=11554717&oldid=11507610
> 

@kelson42 So you did it only for the languages that I explicitly listed in my comment. Currently there are 8 other translations with the same problem (`he`, `lb`, `mk`, `nl`, `nqo`, `sc`, `sl`, `zh-hant`).

> @veloman-yunkan My recommendation, is please open an account on TW so you can do it yourself when needed and possible.

I registered with translatewiki but need to be approved as a translator. I don't qualify as a translator for the languages where the syntax of that message must be fixed. 

Another issue that the TW editor seems to mess-up a message that is a mixture of right-to-left and left-to-right scripts. Here is what it looks like for the language `he`:

אין תוצאות. האם `<a href="{{URL}}=">`לאפס את המסנן`</a>`?

Reply:
@veloman-yunkan I have fixed `he`, `lb`, `mk`, `nl`, `nqo`, `sc`, `sl`. `zh-hant` has already been fixed by me the 24th of April.

Approving an account can be pretty long I believe on TW indeed. What is your account name? Have you made a few translations already (this is needed to be approved I believe)? If this is not already done I will work to help you get your account approved.

I understand that you can not know all languages, but here we are talking about technicalities which are language independent. You should be able IMO to do it.

I'm not sure `he` is messed-up honestly. Maybe @amire80 can enlight us here on this?

Reply:
> Approving an account can be pretty long I believe on TW indeed. What is your account name? Have you made a few translations already (this is needed to be approved I believe)? If this is not already done I will work to help you get your account approved.

Actually my request for an account was refused. The translation test made no sense (during registration I didn't  check any languages in addition to English, so the test offered me to translate from English to English) and I perceived it as just a way of familiarizing new users with the UI/translation tools. As a result my "translations" were reduced to copy-pasting the input text with various assistance from the TW UI, and were naturally considered by the admins to bring zero value to the translatewiki database:

> Hi Veloman,
>
> Thank you for applying as a translator at translatewiki.net. I regret to inform you that I have rejected your application, because the quality of your translations did not meet the requirements.
>
> If you think that your application was rejected by mistake, please try to apply again as a translator at translatewiki.net. You can sign up here:
> https://translatewiki.net/
>
> Amire80,
> translatewiki.net staff